modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -801,6 +929,12 @@ private void dropTableLocally(long causalityToken, String name, UUID tblId, List
         }
     }
 
+    private List<ClusterNode> calculateAssignments(TableConfiguration tableCfg, int partNum) {
+        Integer partitions = tableCfg.partitions().value();
```

Comment:
Quick node, it's better when you use "tableCfg.value()" and then access its properties. It's faster and it doesn't involve boxing.

Reply:
Thanks!